### PR TITLE
new: endpoint to kill existing task

### PR DESCRIPTION
**Pull request by IJtLJZ8Rm4Yr**

### CODE REVIEW

File: src/pymir-app/app/api/api_v1/endpoints/tasks.py [hunk 1]
```diff
@@ -280,6 +280,23 @@ def update_task_name(
     return {"result": task}
 
 
+@router.post(
+    "/{task_id}/stop", response_model=schemas.TaskOut,
```
Comment (phoenix-xhuang) [inline src/pymir-app/app/api/api_v1/endpoints/tasks.py:284]: stop -> terminate

File: src/pymir-app/app/utils/ymir_controller.py [hunk 2]
```diff
@@ -207,6 +208,13 @@ def prepare_inference(
         request.model_config = args["config"]
         return request
 
+    def prepare_kill(
```
Comment (phoenix-xhuang) [inline src/pymir-app/app/utils/ymir_controller.py:211]: 这个函数在哪里用到的？

Comment (IJtLJZ8Rm4Yr) [reply src/pymir-app/app/utils/ymir_controller.py:211]: https://github.com/IndustryEssentials/ymir-backend/blob/master/src/pymir-app/app/utils/ymir_controller.py#L52

**Comment by IJtLJZ8Rm4Yr** [comment]
todo: support kill label task

**Review by phoenix-xhuang — APPROVED** [review]

**Review by under-chaos — APPROVED** [review]

**Pull request merged**